src/gfx/gfx.natvis

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what's this?

Reply:
I added a visualizer for the UniqueId type (only works for the Microsoft C++ extensions though).
Thought I might as well add it